dbbackup/settings.py

```diff
@@ -37,11 +37,6 @@
 
 CUSTOM_CONNECTOR_MAPPING = getattr(settings, 'DBBACKUP_CONNECTOR_MAPPING', {})
 
-# Logging
-LOGGING = getattr(settings, 'DBBACKUP_LOGGING', dbbackup.log.DEFAULT_LOGGING)
-LOG_CONFIGURATOR = logging.config.DictConfigurator(LOGGING)
-LOG_CONFIGURATOR.configure()
```

Comment:
These are the problematic lines: they override the user's logging config inside the project's `settings.py`